rma/tests/test_supplier_rma.py

```diff
@@ -136,22 +136,22 @@ def test_supplier_rma(self):
                                   "Wrong qty delivered")
         picking_out.action_assign()
         picking_out.do_transfer()
-        for line in self.rma_supplier_id.rma_line_ids:
-            self.assertEquals(line.qty_to_receive, 0,
+        for line in self.rma_supplier_id.rma_line_ids[0]:
```

Comment:
So if you are checking only the first line, you know the product, so no if clause is needed. I'd iterate over all the lines as it's already implemented.